### ISSUE TITLE
Goodluck, i could not get this to work at all lmoa

### ISSUE THREAD

**Issue by isubasinghe** [khanguslee/capacitor-okta-spike#2]
Lemme know if you solve this, this broke my brain

**Comment by khanguslee** [comment]
Do you have any code you can show me to see how you did it? I'm just killing time whilst waiting for my AWS credentials...

**Comment by isubasinghe** [comment]
No, its all on codecommit actually, maybe you could get someone else to download those repos for you though? Its called capacitor-auth-testing, i think, it has different branches for the different things I tested

**Comment by isubasinghe** [comment]
But tbh, the code wasnt really written with readability or understandabilty in mind

**Comment by khanguslee** [comment]
I don't think this is any better..
